buildSrc/src/main/groovy/com/google/firebase/gradle/plugins/measurement/utils/enums/ColumnName.groovy

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.gradle.plugins.measurement.utils.enums
+
+import com.google.gson.annotations.SerializedName
+
+enum ColumnName {
```

Comment:
I'm not sure if this enum is best. How about an enum that represents each table? It can contain both the table name and the column names:

```
enum TableLayout {
  APK_SIZES("ApkSizes", ["pull_request_id", "sdk_id", "apk_size"]);

  private final String tableName;
  private final String[] columnNames;

  private TableLayout(String tableName, String[] columnNames) {
  }
}
```